src/main/java/org/sonarlint/intellij/util/SonarLintAppUtils.java

```diff
@@ -128,6 +130,15 @@ private static String getPathRelativeToProjectBaseDir(Project project, VirtualFi
     return VfsUtilCore.getRelativePath(file, projectDir);
   }
 
+  @CheckForNull
+  public static VirtualFile findByRelativePath(VirtualFile file, Path path) {
+    return file.findFileByRelativePath(getSystemIndependentPath(path));
+  }
+
+  public static String getSystemIndependentPath(Path filePath) {
```

Comment:
If possible, I would try to make this one private, to not encourage people to use it